pom.xml

```diff
@@ -3051,7 +3051,8 @@
               <arg>-explaintypes</arg>
               <arg>-release</arg>
               <arg>17</arg>
-              <arg>-Wconf:cat=deprecation:wv,any:e</arg>
+              <arg>-Wconf:any:e</arg>
```

Comment:
Perhaps we should add some comments and highlight this change in the PR description.

Reply:
Updated.